src/crimson/osd/ops_executer.h

```diff
@@ -282,6 +282,8 @@ class OpsExecuter : public seastar::enable_lw_shared_from_this<OpsExecuter> {
   bool has_seen_write() const {
     return num_write > 0;
   }
+
+  version_t get_last_user_version() const;
```

Comment:
Is there a business in defining this tiny getter in `.cc`?

Reply:
There is a circular dependency between OpsExecuter and PG, therefore methods accessing PG members must be defined in `.cc`, I added a comment for that now. Thanks!